src/components/Filter/Filter.style.ts

```diff
@@ -0,0 +1,131 @@
+import { css } from '@emotion/core';
+import { darken, rem } from 'polished';
+
+import { Theme } from '../../theme';
+import { stateBackgroundColor } from '../Button/utils';
+import { ButtonStyleProps } from './types';
+import { getBackgroundColor, getTextColor, getBorder } from './utils';
+
+export const wrapperStyle = () => () => css`
+  position: relative;
+  display: inline-block;
+`;
+
+export const buttonSpanStyle = () => () => {
+  return {
+    display: 'flex',
+    alignItems: 'center',
+  };
+};
+
+export const optionsStyle = () => (theme: Theme) => css`
```

Comment:
this is being used on the `menu` component. As this is used in 2 places now can we make a simple one to handle both cases?